### PR TITLE
[WIP] global update

### PR DESCRIPTION
**Pull request by fanatid**
This PR is controversial, mostly I cleaned repository (delete unused files, move all browser related to own folder, move data to JSON files).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,12 @@ node_js:
   - "0.10"
   - "0.11"
   - "0.12"
-  - "iojs"
+  - "4"
+  - "5"
+matrix:
+  include:
+    - node_js: "4"
```
Comment (calvinmetcalf) [inline .travis.yml]: how about just 'node' here

**Comment by calvinmetcalf** [comment]
can we split this into 2 stages

1. renaming and reformatting
2. any changes to actual code

File: .travis.yml (hunk 1 above)
Comment (fanatid) [inline .travis.yml]: sorry, don't understand

**Comment by fanatid** [comment]
of course, I'll do this

**Comment by fanatid** [comment]
Done in #25 and #26 :)

**Pull request closed without merging**